plugins/crypto/mbedtls/ua_pki_mbedtls.c

```diff
@@ -471,6 +477,29 @@ certificateVerification_verify(void *verificationContext,
         } else {
             retval = UA_STATUSCODE_BADSECURITYCHECKSFAILED;
         }
+
+#ifdef UA_ENABLE_CERT_REJECTED_DIR
+        if(ci->rejectedListFolder.length !=0) {
+            char rejectedFileName[256] = {0};
+            UA_ByteString thumbprint;
+            UA_ByteString_allocBuffer(&thumbprint, UA_SHA1_LENGTH);
```

Comment:
Memory must be cleared after usage (i.e. copying) with UA_ByteString_clear.

Reply:
Thanks, I've added the code.